system-addon/test/unit/content-src/components/LinkMenu.test.js

```diff
@@ -4,26 +4,36 @@ const {_unconnected: LinkMenu} = require("content-src/components/LinkMenu/LinkMe
 const ContextMenu = require("content-src/components/ContextMenu/ContextMenu");
 
 describe("<LinkMenu>", () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallowWithIntl(<LinkMenu site={{url: ""}} dispatch={() => {}} />);
+  });
   it("should render a ContextMenu element", () => {
-    const wrapper = shallowWithIntl(<LinkMenu />);
     assert.ok(wrapper.find(ContextMenu));
   });
   it("should pass visible, onUpdate, and options to ContextMenu", () => {
-    const wrapper = shallowWithIntl(<LinkMenu />);
+    assert.ok(wrapper.find(ContextMenu));
     const contextMenuProps = wrapper.find(ContextMenu).props();
-    ["visible", "onUpdate", "options", "tabbableOptionsLength"].forEach(prop => assert.property(contextMenuProps, prop));
+    ["visible", "onUpdate", "options"].forEach(prop => assert.property(contextMenuProps, prop));
   });
   it("should give ContextMenu the correct tabbable options length for a11y", () => {
-    const wrapper = shallowWithIntl(<LinkMenu />);
+    const firstItemLabel = "item1";
+    const lastItemLabel = "item2";
 
     // stub the getOptions method
     let component = wrapper.instance();
-    component.getOptions = () => [{label: "item1"}, {type: "separator"}, {label: "item2"}];
+    component.getOptions = () => [{label: firstItemLabel}, {type: "separator"}, {label: lastItemLabel}];
 
     // force the component and wrapper to update so that the stub is used
     component.forceUpdate();
     wrapper.update();
-    const contextMenuProps = wrapper.find(ContextMenu).props();
-    assert.equal(contextMenuProps.tabbableOptionsLength, 2);
+
+    const options = component.getFirstAndLast(component.getOptions());
+    const firstItem = options[0];
+    const lastItem = options[options.length - 1];
+    assert.equal(firstItem.label, firstItemLabel);
+    assert.isTrue(firstItem.first);
+    assert.equal(lastItem.label, lastItemLabel);
+    assert.isTrue(lastItem.last);
```

Comment:
A negative test that the first item `isTrue(!firstItem.last)`, and similarly for last, but also that the middle item has neither.